yt_dlp/extractor/agora.py

```diff
@@ -0,0 +1,266 @@
+            |wysokieobcasy\.pl/wysokie-obcasy/0,176631\.html)
+        (?:\?(?:[^&]+?&)*?podcast=(?P<id>\d+))?
+    '''
+    _TESTS = [{
+        'url': 'https://wyborcza.pl/podcast/0,172673.html?podcast=100720#S.main_topic-K.C-B.6-L.1.podcast',
+        'info_dict': {
+            'id': '100720',
+            'ext': 'mp3',
+            'title': 'Cyfrodziewczyny. Kim były pionierki polskiej informatyki ',
+            'uploader': 'Michał Nogaś ',
+            'upload_date': '20210117',
+            'description': 'md5:49f0a06ffc4c1931210d3ab1416a651d',
+            'duration': 3684.0,
+            'thumbnail': r're:https://.+\.jpg',
+        },
+    }, {
+        'url': 'https://www.wysokieobcasy.pl/wysokie-obcasy/0,176631.html?podcast=100673',
+        'info_dict': {
+            'id': '100673',
+            'ext': 'mp3',
+            'title': 'Czym jest ubóstwo menstruacyjne i dlaczego dotyczy każdej i każdego z nas?',
+            'uploader': 'Agnieszka Urazińska ',
+            'upload_date': '20210115',
+            'description': 'md5:c161dc035f8dbb60077011fc41274899',
+            'duration': 1803.0,
+            'thumbnail': r're:https://.+\.jpg',
+        },
+    }, {
+        'url': 'https://wyborcza.pl/podcast',
+        'info_dict': {
+            'id': '334',
+            'title': 'Gościnnie: Wyborcza, 8:10',
+            'series': 'Gościnnie: Wyborcza, 8:10',
+        },
+        'playlist_mincount': 370,
+    }, {
+        'url': 'https://www.wysokieobcasy.pl/wysokie-obcasy/0,176631.html',
+        'info_dict': {
+            'id': '395',
+            'title': 'Gościnnie: Wysokie Obcasy',
+            'series': 'Gościnnie: Wysokie Obcasy',
+        },
+        'playlist_mincount': 12,
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        podcast_id = mobj.group('id')
+
+        # linking to the playlist and not specific episode
+        if not podcast_id:
+            return {
+                '_type': 'url',
+                'url': 'tokfm:audition:%s' % ('395' if 'wysokieobcasy.pl/' in url else '334'),
+                'ie_key': 'TokFMAudition',
+            }
+
+        meta = self._download_json('https://wyborcza.pl/api/podcast?guid=%s%s' % (podcast_id,
+                                                                                  '&type=wo' if 'wysokieobcasy.pl/' in url else ''),
+                                   podcast_id)
+        published_date = meta['publishedDate'].split(' ')
+        upload_date = '%s%s%s' % (published_date[2], {
+            'stycznia': '01',
+            'lutego': '02',
+            'marca': '03',
+            'kwietnia': '04',
+            'maja': '05',
+            'czerwca': '06',
+            'lipca': '07',
+            'sierpnia': '08',
+            'września': '09',
+            'października': '10',
+            'listopada': '11',
+            'grudnia': '12',
+        }.get(published_date[1]), ('0' + published_date[0])[-2:])
```

Comment:
use utils.month_by_name

Reply:
Also, this should be non-fatal

Reply:
utils.month_by_name has no support for polish

Reply:
you can add these names to it instead of defining it here